controllers/networkchaos/types.go

```diff
@@ -41,16 +48,54 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		return ctrl.Result{}, nil
 	}
 
+	scheduler := networkchaos.GetScheduler()
+	duration, err := networkchaos.GetDuration()
+	if err != nil {
+		r.Log.Error(err, fmt.Sprintf("unable to get podchaos[%s/%s]'s duration", networkchaos.Namespace, networkchaos.Name))
+		return ctrl.Result{}, nil
+	}
+	if scheduler == nil && duration == nil {
```

Comment:
what if duration != nil

Reply:
@zhouqiang-cl `Networkchaos` controller only handle the situation when `scheduler` and `duration` are both defined or omitted. For the other situation, it would log the error and return the empty response.

More over, I also think Adding validation admission webhook for each chaos kind to check the `scheduler` and `duration` situation is necessary as I mentioned in #171

Reply:
Got